package concurrency.in.practice.exception;


public class TestException extends RuntimeException {

    
}
